src/Invoice/Asset/core/scss/_core.scss

```
@@ -16,12 +16,20 @@
 
 html {
   min-height: 100vh;
+  // Moved here from body: a `position: sticky` element's containing block
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)